{{cookiecutter.project_slug}}/{{cookiecutter.project_slug}}/templates/403.html

```diff
@@ -5,5 +5,5 @@
 {% block content %}
 <h1>Forbidden (403)</h1>
 
-<p>CSRF verification failed. Request aborted.</p>
+<p>{% if exception %}{{ exception }}{% else %}CSRF verification failed. Request aborted.{% endif %}</p>
```

Comment:
Let's fix this confusing message, 403 might happen in other cases than just CSRF error:

```suggestion
<p>{% if exception %}{{ exception }}{% else %}You're not allowed to access this page.{% endif %}</p>
```

Reply:
Just double checking: if a user submits a form without the CSRF token, then the exception variable should evaluate true right?

Reply:
I believe it doesn't. I'm somewhat concerned that a dev wouldn't know if a form needed a CSRF token because this exception page has helped me in the past.

This is in the CSRFMiddleware:

```python
    def _reject(self, request, reason):
        response = _get_failure_view()(request, reason=reason)
        log_response(
            'Forbidden (%s): %s', reason, request.path,
            response=response,
            request=request,
            logger=logger,
        )
        return response
```
though via the `log_response` the dev can see in the console that they need a CSRF token.

But I think this is OK!